modules/cms/controllers/ThemeOptions.php

```diff
@@ -143,4 +150,20 @@ protected function findThemeObject($name = null)
 
         return $theme;
     }
+
+    /**
+     * Register language namespace if available for the theme
+     *
+     * @param string $dirName
+     * @throws ApplicationException
```

Comment:
@GinoPane As with https://github.com/octobercms/october/pull/4972, if we are going to start adding **@throws** declarations in the docs, we need to specify some context as to *why* they are thrown. Would you mind adding in an example of why `ApplicationException` would be thrown for this method?

Reply:
@bennothommo do you mean adding it into a docblock here? Basically the exception is inherited from findThemeObject method where it is thrown when the theme name is not passed via POST for any reason or when the theme cannot be loaded.

Reply:
Like 
```
@throws ApplicationException if the theme object cannot be instantiated
```

Reply:
@GinoPane sorry for the delay in responding - yes, please add the example in this docblock.

Reply:
@bennothommo done!